internal/daysteps/daysteps.go

```diff
@@ -1,15 +1,42 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
 	"time"
+
+	spentСalories "github.com/Yandex-Practicum/go1fl-4-sprint-final/internal/spentcalories"
 )
 
+const timeFormat = "15:04"
+
 var (
 	StepLength = 0.65 // длина шага в метрах
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// ваш код ниже
+	row := strings.Split(data, ",")
+	if len(row) != 2 {
+		return 0, 0, errors.New("Slice != two")
+	}
+	steps, err := strconv.Atoi(row[0])
+	if err != nil {
+		return 0, 0, err
```

Comment:
Лучше оборачивать ошибки, чтобы был контекст
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```